core/dbt/events/proto_types.py

```diff
@@ -2016,6 +2016,118 @@ class RunResultWarningMessage(betterproto.Message):
     msg: str = betterproto.string_field(2)
 
 
+@dataclass
+class DebugEnvironmentDetails(betterproto.Message):
```

Comment:
We have been going through an exercise of combining various logging events. These new logging events are pretty much identical except for the names and codes. I think it would be better to combine some of these messages. I would personally prefer a sort of "DebugCmdResult" message. Or maybe some of those could be put in a DebugCmdOut event.  The touchstone is to think of what would be most convenient for someone who's looking for particular information in the logs.

Reply:
Did my best with this!